packages/kit/src/views/Onboardingv2/pages/ICloudBackup.tsx

```
@@ -44,6 +48,14 @@ export default function ICloudBackup() {
   const [refreshHook] = useOnboardingCloudBackupListRefreshAtom();
   const intl = useIntl();
 
+
+  const { result: legacyBackups } = usePromiseResult(async () => {
```

ESLint

[error] Delete `⏎`

(prettier/prettier)